packages/create-neutrino-project/bin/create-neutrino-project.js

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env node
```

Comment:
I'm thinking the syntax of this command should be:

```js
create-neutrino-project <directory>
```

So that we know the directory name to create, as well as a project name to infer from this directory. We should throw when this option is not preset. That will also save us from asking the project name question.